.eslintrc.cjs

```diff
@@ -1,19 +1,27 @@
 /* eslint-env node */
 module.exports = {
-extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
-  parser: '@typescript-eslint/parser',
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  parser: "@typescript-eslint/parser",
   parserOptions: {
     ecmaVersion: 13,
-    project: './tsconfig.json',
-    sourceType: 'module',
+    project: "./tsconfig.eslint.json",
+    tsconfigRootDir: __dirname,
+    sourceType: "module",
   },
-  plugins: ['@typescript-eslint'],
+  overrides: [
+    {
+      files: ["test/**/*.{ts,tsx}"],
+      parserOptions: {
+        project: "./tsconfig.test.json",
+      },
+    },
+  ],
+  plugins: ["@typescript-eslint"],
   rules: {
-    'import/prefer-default-export': 'off',
-    'react/require-default-props': 'off',
-    '@typescript-eslint/no-use-before-define': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
Can we keep` '@typescript-eslint/no-explicit-any': 'off',`? Just to avoid this kind of messages

<img width="1035" alt="image" src="https://github.com/PADAS/react-native-jsonforms-formatter/assets/87322013/1c39cf1d-0ec6-4e8e-9c13-b798d9e5fbfd">

Reply:
Yes, added it back, would like to reduce the usage but we can consider down the road